cmd/helpers.go

```diff
@@ -53,28 +54,48 @@ func InteractiveRolePrompt(args []string, region string, client *creds.Client) (
 	}
 
 	// Retrieve the list of roles
-	roles, err := client.Roles()
+	rolesExtended, err := client.RolesExtended()
 	if err != nil {
 		return "", err
 	}
+	var roles []string
+	var rolesSearch []string
+	maxLen := 12
+	for _, role := range rolesExtended {
+		if len(role.AccountName) > maxLen {
+			maxLen = len(role.AccountName)
+		}
+	}
+	maxLenS := strconv.Itoa(maxLen)
+	for _, role := range rolesExtended {
+		account := role.AccountName
+		if account == "Unknown" {
+			account = role.AccountNumber
+		}
+		account = fmt.Sprintf("%-"+maxLenS+"s", account)
+		roles = append(roles, account+"\t"+role.RoleName)
+		// So users can search <account friendly name> <role> or <role> <account friendly name>
+		rolesSearch = append(rolesSearch, role.AccountName+role.Arn+role.AccountName)
+	}
 
 	// Prompt the user
 	prompt := promptui.Select{
-		Label: "Select Role",
+		Label: "You can search for role name or account name/number or a combination of the two, e.g. prod appname",
 		Items: roles,
 		Size:  16,
```

Comment:
Can you bump this down to 10? 16 is a little busy/crowded imo

Reply:
Yup, done!